schema/rdl_schema_0.1.json

```
@@ -48,20 +50,24 @@
 			"format": "date"
 		},
 		"version": {
+			"title": "Dataset version",
 			"type": "integer",
-			"description": "Version of the dataset"
+			"description": "The version of the dataset."
 		},
 		"purpose": {
+			"title": "Dataset purpose",
 			"type": "string",
-			"description": "Purpose for what the data has been produced"
+			"description": "The purpose the dataset was produced for."
```

Copy edit to use more formal language:

```suggestion
			"description": "The purpose for which the dataset was created."
```